fe/fe-core/src/main/java/org/apache/doris/backup/BackupJob.java

```diff
@@ -169,6 +169,55 @@ public BackupContent getContent() {
         return BackupContent.ALL;
     }
 
+    private synchronized boolean tryNewTabletSnapshotTask(SnapshotTask task) {
+        Table table = env.getInternalCatalog().getTableByTableId(task.getTableId());
+        if (table == null) {
+            return false;
+        }
+        OlapTable tbl = (OlapTable) table;
+        if (tbl.getId() != task.getTableId()) {
+            return false;
+        }
+        Partition partition = tbl.getPartition(task.getPartitionId());
```

Comment:
Here we need to add a read lock for OlapTable

Reply:
done